framework/include/utils/ValueCache.h

```diff
@@ -28,32 +41,36 @@ class ValueCache
 {
 public:
   ValueCache(std::size_t in_dim, std::size_t max_leaf_size = 10);
+  ValueCache(const std::string & file_name, std::size_t in_dim, std::size_t max_leaf_size = 10);
```

Comment:
Can you please add just a little docstring action here

Reply:
done